doc/2/controllers/document/search/snippets/search-koncorde.dart

```diff
@@ -0,0 +1,9 @@
+final result = await kuzzle
+  .document
+  .search('nyc-open-data', 'yellow-taxi', query: {
+    'query': {
```

Comment:
I'm not sure the top level `query` is needed. I rather think that this should raise an error instead (Kuzzle do raise an error)

Reply:
As we discussed, this is normal here